MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,364 @@
+# Migration Guide
+
+## Migrate from OpenTracing to OpenTelemetry 
+
+### Starter Bundle
+If you do not use sda-commons-starter with [SdaPlatformBundle](../../sda-commons-starter/src/main/java/org/sdase/commons/starter/SdaPlatformBundle.java), you need to remove the Jaeger bundle and OpenTracing bundle and add the OpenTelemetry bundle.
+
+Before:
+```java
+bootstrap.addBundle(JaegerBundle.builder().build());
+bootstrap.addBundle(OpenTracingBundle.builder().build());
+```
+
+After:
+```java
+bootstrap.addBundle(
+        OpenTelemetryBundle.builder()
+            .withAutoConfiguredTelemetryInstance()
+            .withExcludedUrlsPattern(Pattern.compile(String.join("|", excludedTracingUrls)))
+            .build());
+```
+
+### Replace OpenTracing dependencies to OpenTelemetry dependencies
+
+Before:
+```gradle
+testImplementation 'io.opentracing:opentracing-mock'
+```
+
+After:
+```gradle
+testImplementation 'io.opentelemetry:opentelemetry-sdk-testing'
+```
+
+### Executors need to be instrumented to follow traces when parts of the request are handled asynchronously.
+
+Before:
+```java
+return new AsyncEmailSendManager(
+  new InMemoryAsyncTaskRepository<>(limits.getMaxAttempts()),
```

Comment:
Can we give some advice for `limits.getMaxAttempts()`?

Reply:
This is just a parameter for the creation of `InMemoryAsyncTaskRepository`. It has nothing to do with open telemetry. The important part is wrapping the async task inside a open telemetry Context

Reply:
Ah, sorry. I overread that this is the "Before" section. I can see it in the "After" section. 👍